test/e2e/saucelabs.conf.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-console, no-process-env */
+
+const config = require('config');
+const fileAcceptor = require('test/file_acceptor');
+const supportedBrowsers = require('./crossbrowser/supportedBrowsers.js');
+
+const evidenceUploadEnabled = config.get('features.evidenceUpload.enabled');
+const tunnelName = process.env.SAUCE_TUNNEL_IDENTIFIER || config.get('saucelabs.tunnelId');
+
+const getBrowserConfig = browserGroup => {
+  const browserConfig = [];
+  for (const candidateBrowser in supportedBrowsers[browserGroup]) {
+    if (candidateBrowser) {
+      const desiredCapability = supportedBrowsers[browserGroup][candidateBrowser];
+      desiredCapability.tunnelIdentifier = tunnelName;
+      desiredCapability.tags = ['sscs'];
+      browserConfig.push({
+        browser: desiredCapability.browserName,
+        desiredCapabilities: desiredCapability
+      });
+    } else {
+      console.error('ERROR: supportedBrowsers.js is empty or incorrectly defined');
+    }
+  }
+  return browserConfig;
+};
+
+const setupConfig = {
+  tests: './smoke/appeal.test.js',
+  output: process.env.E2E_OUTPUT_DIR || config.get('e2e.outputDir'),
+  features: {
+    evidenceUpload: {
+      enabled: evidenceUploadEnabled
+    }
+  },
+  helpers: {
+    WebDriverIO: {
```

Comment:
cannot we use Puppeteer?

Reply:
No unfortunately not, Saucelabs needs a remote webdriver instance to tell it what commands to execute, whereas puppeteer is a tool for Chrome/Chromium only